e2e/common/common.steps.ts

```diff
@@ -86,6 +102,27 @@ class CommonSteps {
       .to.eventually.be.true.notify(callback);
   }
 
+  @then(/^she is presented with the "([^"]*)" elements*$/)
+  public expectElementsPresent(elementClassNames: string, callback: CallbackStepDefinition): void {
+
+    const elementClassNamesArray = elementClassNames.split(',');
```

Comment:
@lvydra I would be better to look up the table by title like `Integration Board`, `Recent Updates` etc. to avoid specific element declaration in steps file. Could you please take a look?